html/Makefile.in

```diff
@@ -34,17 +29,16 @@ clean:
 	rm -f docs/images/*.jbf
 	rm -f stylesheets/*~
 	rm -f js/*~
-	rm -rf angularjs/angular-1.3.9
-	rm -rf angularjs/ui-utils-0.2.3
-	rm -rf bootstrap-3.3.7
-	rm -rf d3
 
 distclean: clean
 	rm -f Makefile config.inc.php
 
 devclean: distclean
 
 install:
+	# Remove old files
+	rm -f $(DESTDIR)$(HTMLDIR)/*
```

Comment:
Is this an okay change? I'm not sure what else would get put in this directory. The only thing I could think of that we would want to preserve is the config.inc.php file but that gets overwritten by the install command.

Reply:
My concern is if people regularly put other things in this file that we don't want deleted. I can always specify the exact files we want deleted anyway.

Reply:
yeah no, I'm not in favor of that. I know it looks gross but I prefer the previous style

Reply:
Alright, updated